### PR TITLE
[full-ci][tests-only] Remove @yetToImplement tags from tests (part4)

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
remove @yetToImplement from tests and add them to expected-failures if required features not implemented yet

## Related Issue
- part of 
https://github.com/owncloud/web/issues/4977

## How Has This Been Tested?
- test environment:

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:
- [ ] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingInternalUsers2</strong> https://drone.owncloud.com/owncloud/web/16967/53/1
The following scenarios passed on retry:
- webUISharingInternalUsersShareWithPage/shareWithUsers.feature:136

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16967/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10Basic</strong> https://drone.owncloud.com/owncloud/web/16967/6/1
<details><summary>:boom: The acceptance tests failed. Please find the screenshots inside ...</summary>

<p>


### webUILogin-adminBlocksUser-feature-20.png
  

### webUIPreview-imageMediaViewer-feature-141.png
  

### webUIPreview-imageMediaViewer-feature-159.png
  

### webUIPreview-imageMediaViewer-feature-84.png
  

### webUIPreview-imageMediaViewer-feature-91.png
  

### webUIWebdavLockProtection-delete-feature-75.png
  

### webUIWebdavLockProtection-delete-feature-76.png
  

### webUIWebdavLockProtection-move-feature-146.png
  

### webUIWebdavLockProtection-move-feature-147.png
  

### webUIWebdavLockProtection-upload-feature-90.png
  

### webUIWebdavLockProtection-upload-feature-91.png
  


</p></details>

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/16987/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingAccept</strong> https://drone.owncloud.com/owncloud/web/16987/9/1
The following scenarios passed on retry:
- webUISharingAcceptShares/acceptShares.feature:99

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17055/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17056/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17100/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by phil-davis** [comment]
@saw-jan there are now conflicts - some other PRs have been merged to web master.

**Review by SwikritiT — COMMENTED** [review]
LGTM

File: tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md [hunk 1]
```diff
@@ -191,4 +196,23 @@ Other free text and markdown formatting can be used elsewhere in the document if
...
+-   [webUIUpload/upload.feature:121](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/upload.feature#L121)
+-   [webUIUpload/upload.feature:138](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUIUpload/upload.feature#L138)
+
+### [Federated shares not showing in shared with me page] (https://github.com/owncloud/web/issues/2510)
+-   [webUISharingExternalToRoot/federationSharing.feature:32](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingExternalToRoot/federationSharing.feature#L32)
+-   [webUISharingExternalToRoot/federationSharing.feature:167](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingExternalToRoot/federationSharing.feature#L167)
+-   [webUISharingExternal/federationSharing.feature:38](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingExternal/federationSharing.feature#L38)
+-   [webUISharingExternal/federationSharing.feature:166](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingExternal/federationSharing.feature#L166)
+
+### [Browse directly to sidebar options]
```
Comment (kiranparajuli589) [inline tests/acceptance/expected-failures-with-oc10-server-oauth2-login.md:212]: no issue? 🤔

File: tests/acceptance/features/webUIUpload/upload.feature [hunk 2]
```diff
@@ -133,7 +134,7 @@ Feature: File Upload
     Then file "lorem.txt" should be listed on the webUI
     And as "Alice" the content of "simple-folder/lorem.txt" should be the same as the local "lorem.txt"
 
-  @skip @yetToImplement
+  @issue-5106 @ocis-reva-issue-64
```
Comment (kiranparajuli589) [inline tests/acceptance/features/webUIUpload/upload.feature:137]: do not add the closed issue

Comment (saw-jan) [reply tests/acceptance/features/webUIUpload/upload.feature:137]: `@ocis-reva-issue-64` this one?

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17112/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17115/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

File: tests/acceptance/features/webUIUpload/upload.feature (hunk 2 above)
Comment (kiranparajuli589) [reply tests/acceptance/features/webUIUpload/upload.feature]: yes

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17180/49/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10CreateDelete</strong> https://drone.owncloud.com/owncloud/web/17180/8/1
The following scenarios passed on retry:
- webUIDeleteFilesFolders/deleteFilesFolders.feature:59

**Comment by ownclouders** [comment]
Results for <strong>oCISBasic</strong> https://drone.owncloud.com/owncloud/web/17219/46/1
The following scenarios passed on retry:
- webUIPreview/imageMediaViewer.feature:15

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingExternal</strong> https://drone.owncloud.com/owncloud/web/17231/38/1
The following scenarios passed on retry:
- webUISharingExternal/federationSharing.feature:364

**Review by phil-davis — APPROVED** [review]

**Pull request merged**